helix-view/src/keyboard.rs

```diff
@@ -155,9 +157,9 @@ impl From<crossterm::event::KeyCode> for KeyCode {
             | CKeyCode::Menu
             | CKeyCode::KeypadBegin
             | CKeyCode::Media(_)
-            | CKeyCode::Modifier(_) => unreachable!(
+            | CKeyCode::Modifier(_) => bail!(
                 "Shouldn't get this key without enabling DISAMBIGUATE_ESCAPE_CODES in crossterm"
```

Comment:
```suggestion
                "Shouldn't get key {:?} without enabling DISAMBIGUATE_ESCAPE_CODES in crossterm",
                val
```

Since the unreachable doesn't point at this line anymore, this should make it easier to debug if it shows up in the logfile